src/main/java/org/apache/maven/shared/utils/cli/shell/Shell.java

```diff
@@ -29,12 +29,11 @@
 /**
  * <p>
  * Class that abstracts the Shell functionality,
- * with subclases for shells that behave particularly, like
+ * with subclasses for shells that behave particularly, like</p>
```

Comment:
Javadoc 8 will probably bug you for this. They expect `<p>` only.

Reply:
fixed